docs/guidelines/accessibility.md

```diff
@@ -1,3 +1,659 @@
+
+### 2. Build positive public relations 
+
+An organization's efforts in Web accessibility are a public relations opportunity to increase its positive image, which can increase website use.
+  
+Corporate Social Responsibility (CSR) has been shown to improve financial performance, enhance brand image and reputation, increase sales and customer loyalty, increase ability to attract and retain employees, and provide access to capital and funding. Additional perspectives on CSR, such as statistics that show how CSR impacts customers, are available on the Web.
+
+### 3. Broaden your market penetration
+
+You expand your customer/audience base. According to a [2010 report from the U.S. Census Bureau](https://www.census.gov/content/dam/Census/library/publications/2012/demo/p70-131.pdf), around 18.7% of the U.S. population lives with a disability and, out of that number, 12.6% have a severe disability.  
+
+A [study in the United Kingdom (UK)](https://disability-smart.com/2017/01/10/is-there-really-a-business-case-for-website-accessibility/) found that 71% of web users with a disability will simply leave a website that is not accessible. 
+
+    
+### 4. Avoid discrimination and legal complications
+
+Your website will comply with your country’s equality and anti-discrimination law.
+  
+Internationally, governments, but also private companies and organisations, are becoming subject to legal challenges because of their inaccessible online content and service offerings.
+
+In the US, public sector entities (government-funded programs/schools, airlines, and nonprofits) are required to follow certain website accessibility rules, while private sector companies just [hope they will not get sued](https://insight.cryptzone.com/accessibility/6-web-accessibility-takeaways-winn-dixie-accessibility-decision/).
+  
+In 2017 the Information and Communication Technology (ICT) passed a ruling to update the old 508 rules to be more inline with WCAG 2.0 guidelines, meaning that website accessibility updates are required for most public websites — and it is just a matter of time before private websites will be as well.
+
+### 5. Better Search Engine Optimization (SEO)  
+
+Accessibility techniques increase the findability of web pages by exposing content to search engines, both internally (within a website) and externally (across the World Wide Web).  Most of the search engines are only able to process text and the way it is structured.  By having a well-marked up site, they can index your site for more relevant information in search results. 
+
+
+### 6. Improve usability for all types of users
```

Comment:
some of these could be repetitive from the list above. but I like listing out all the users that accessibility benefits. I would recommend have this list closer to the list of types of conditions --> https://github.com/massgov/edit-mayflower-docs/pull/4/files#diff-501a3ff63789efdf5a9eecb360697a23R239